### PR TITLE
Support the assumes keyword

### PR DESCRIPTION
**Pull request by sergiusens**
LP: #1586429

Signed-off-by: Sergio Schvezov <sergio.schvezov@ubuntu.com>

### CODE REVIEW

**Comment by sergiusens** [comment]
@pedronis a sanity check from your side would be ideal

File: docs/snapcraft-syntax.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ contain.
   The type of confinement supported by the snap. Can be either "devmode" (i.e.
   this snap doesn't support running under confinement) or "strict" (i.e. full
   confinement supported via interfaces).
+* `assumes` (list of strings)
+  A feature tag list that the core needs to support for this snap to be run.
```
Comment (kyrofa) [inline docs/snapcraft-syntax.md]: From the bug, this seems to be an install-time check. Maybe "A list of features that must be supported by the core in order for this snap to install." ?

Comment (sergiusens) [inline docs/snapcraft-syntax.md]: El 27/05/16 a las 13:02, Kyle Fazzari escribió:
> In docs/snapcraft-syntax.md
> <https://github.com/ubuntu-core/snapcraft/pull/525#discussion_r64927675>:
>
> > @@ -21,6 +21,8 @@ contain.
> >    The type of confinement supported by the snap. Can be either "devmode" (i.e.
> >    this snap doesn't support running under confinement) or "strict" (i.e. full
> >    confinement supported via interfaces).
> > +* `assumes` (list of strings)
> > +  A feature tag list that the core needs to support for this snap to be run.
>
> From the bug, this seems to be an install-time check. Maybe "A list of
> features that must be supported by the core in order for this snap to
> install." ?
Good call

**Comment by kyrofa** [comment]
Looks good! :+1:

**Pull request merged**